pyproject.toml

```diff
@@ -572,7 +572,6 @@ module = [
     "sentry.nodestore.django.models",
     "sentry.nodestore.filesystem.backend",
     "sentry.nodestore.models",
-    "sentry.profiles.*",
```

Comment:
this is the _stronger_ list -- not the ignored list!

Reply:
ah, my bad @asottile-sentry !

Thanks!